frontend/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Form from "../Form/Form.js";
+import Input from "../Input/Input.js";
+import Button from "../Button/Button.js";
+import "./Footer.css";
+
+import roundel from "../../res/logo_stamp.svg";
+
+const randomItem = array => {
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+function Footer(props) {
+  const phrase = randomItem(props.json.phrases);
+
+  return (
+    <footer>
+      <div className="container">
+        <img src={roundel} alt="Cutie Club" />
+        <div className="wrapper">
+          <div className="left">
+            <h2>{phrase}</h2>
```

Comment:
```suggestion
            <h2>{randomItem(props.json.phrases)}</h2>
```
could do this not sure whats prefferential